### DISCUSSION TITLE
What actually is the Secret api

### DISCUSSION THREAD

**Discussion by 0xBanditt** [ArugaZ/whatsapp-bot#442] [General]
Which secret api is the author talking about?

**Comment by Cosm1cBug** [comment]
It's a secret. 😂🤭

**Comment by arugaz** [comment]
Well that for Fastify backend [here](https://github.com/arugaz/whatsapp-bot/blob/3259843f3109855dd09fd2a320e704a2b3ee6199/src/libs/server/routes.ts#L9)
